packages/link-list/src/LinkListItem.tsx

```diff
@@ -0,0 +1,35 @@
+import { Box } from '@ag.ds-next/box';
+import { useLinkComponent, themeVars, outline } from '@ag.ds-next/core';
+
+export type LinkListItemType = {
+	href: string;
+	label: string;
```

Comment:
```suggestion
	label: ReactNode;
```

so you can pass in an icon along side the component